ui/components/reservation/EditStep0.tsx

```diff
@@ -187,8 +187,8 @@ const EditStep0 = ({
       getSlotPropGetter(
         reservationUnit.openingHours?.openingTimes,
         activeApplicationRounds,
-        reservationUnit.reservationBegins,
-        reservationUnit.reservationEnds,
+        new Date(reservationUnit.reservationBegins),
+        new Date(reservationUnit.reservationEnds),
```

Comment:
I think you fixed this before the function was refactored, but these values also need truthiness check.